lib/temporal/client.rb

```diff
@@ -143,6 +143,14 @@ def describe_namespace(name)
       connection.describe_namespace(name: name)
     end
 
+    # Fetches all the namespaces.
+    #
+    # @param page_size [Integer] number of namespace results to return per page.
+    # @param next_page_token [String] page token to advancing the pagination (provide this )
```

Comment:
Nit: this comment looks unfinished, maybe something like "a optional pagination token returned by a previous list_namespaces call"

Reply:
Good catch, fixed! :) thanks @antstorm